app/main.py

```diff
@@ -1,2 +1,77 @@
+from typing import Hashable, Any
+
+DEFAULT_CAPACITY = 8
```

Comment:
theis should be class variables inside of Dictionary since it is part of it implementation